src/sciwyrm/main.py

```diff
@@ -59,3 +76,7 @@ async def format_notebook(
     nb = json.loads(formatted.body)
     nb["metadata"]["sciwyrm"] = notebook.notebook_metadata(spec)
     return JSONResponse(nb)
+
+
+if __name__ == "__main__":
+    uvicorn.run(app, host="0.0.0.0", port=8000)
```

Comment:
What is this good for? I don't like that it has a fixed address and port. Doesn't this just lead to conflicts?

Reply:
It is the typical way to run  it standalone in dev, but yes making the port configurable would be a nice addition

Reply:
Is it? Judging by the documentation, you are supposed to use `uvicorn sciwyrm.main:app`. Plus, this way, you can use `uvicorn sciwyrm.main:app --reload` which auto reloads the app when files are changed.